app/js/actions/plugins.js

```diff
@@ -52,9 +52,12 @@ export const loadPlugins = () => {
         const url = `http://${uri}/api/plugins/`;
         const method = 'GET';
         fetchAPI(secretKey, method, url)
-        .then(json => json.plugins.map(plugin => dispatch(foundPlugin(plugin))))
+        .then(json => json.plugins.map(plugin => {
+            dispatch(foundPlugin(plugin));
+            return plugin;
```

Comment:
`dispatch()` stopped returning the action once we used the `redux-electron-store` enhancer, so that's what this change is repairing.